server/models/profile.js

```diff
@@ -0,0 +1,56 @@
+const mongoose = require("mongoose");
+const avaliableTime = new mongoose.Schema({
+    avaliableDay: {
+        type: Date,
+        required: true,
+    },
+    timeTo: {
+        type: String,
+        required: true,
+    },
+    timeFrom: {
+        type: String,
+        required: true
+    },
+})
+const profileSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: true,
+    },
+    lastName: {
+        type: String,
+        required: true,
+    },
+    description: {
+        type: String,
+        required: true,
+    },
+    phoneNo: {
+        type: Number,
+        required: true,
+    },
+    photoUrl: {
+        type: String,
+        required: true,
+    },
+    email: {
```

Comment:
Let's remove the `email` field as it already exists within the User document

Reply:
Done the changes